hostfactory/1.1/providerplugins/azurecc/scripts/src/cyclecloud_provider.py

```diff
@@ -617,17 +615,20 @@ def _create_status(self, input_json, json_writer=None):
         
         nodes_by_request_id = {}
         exceptions = []
+        logger.debug(request_ids)
```

Comment:
not sure you want this

Reply:
yes had added for debugging will remove it